src/Handler/ContainerHandler.php

```diff
@@ -26,9 +26,9 @@ class ContainerHandler implements AfterMethodCallAnalysisInterface, AfterClassLi
     private const GET_CLASSLIKES = [
         'Psr\Container\ContainerInterface',
         'Symfony\Component\DependencyInjection\ContainerInterface',
+        'Symfony\Component\DependencyInjection\Container',
         'Symfony\Bundle\FrameworkBundle\Controller\AbstractController',
         'Symfony\Bundle\FrameworkBundle\Controller\ControllerTrait',
-        'Symfony\Bundle\FrameworkBundle\Test\TestContainer',
```

Comment:
we can keep this one as well, right?

Reply:
Sure, but I decided to remove that which is no longer needed.

Let me know if you want to keep it.

Reply:
lets keep it

Reply:
Done.